cadwyn/structure/versions.py

```diff
@@ -446,7 +446,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
+            # None is a valid response body, when a user returns just a status code
+            # dumping it will result in a 'null' string, which doesnt correlate with the content-length
```

Comment:
Let's make it consistent with `main`'s comments.

Reply:
fixed